setup.py

```diff
@@ -49,7 +49,10 @@ def get_install_requires():
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'openwisp-controller>=0.3.1,<0.4.1'
+        'celery>=4.4.0,<5.0',
+        'openwisp-controller>=0.5.1,<0.6.1',
+        'openwisp-utils>=0.4.4,<0.5.0',
+        'djangorestframework>=3.11.0,<3.12'
```

Comment:
why are you adding all these dependencies here? openwisp-controller already depends on those so it should not be needed to re-add those here